### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by fabiodl** [ignc-research/arena-bench#2]
Than you for this extensive work.
I used the command 
`wget https://raw.githubusercontent.com/ignc-research/arena-bench/main/setup.sh -O - | bash`

to install the package on a fresh install of  Ubuntu 20.04.5
The following dependencies required manual installation
`ros-noetic-mbf-costmap-core ibsuitesparse-dev ros-noetic-libg2o ros-noetic-mbf-costmap-core ros-noetic-mbf-msgs coinor-libipopt-dev ros-noetic-dwa-local-planner` 
 
On execution, of the example 
`roslaunch arena_bringup start_arena_gazebo.launch local_planner:=dwa world:=map2 model:=turtlebot3_burger scenario_file:=map2_obs05.json`

I still get the following errors

`ERROR: cannot launch node of type [eval/viz_path.py]: eval
ROS path [0]=/opt/ros/noetic/share/ros
ROS path [1]=/home/fabio/catkin_ws/src
ROS path [2]=/opt/ros/noetic/share`

`[ERROR] [1670399102.217173113]: PluginlibFactory: The plugin for class 'rviz_plugins/Goal3DTool' failed to load.  Error: According to the loaded plugin descriptions the class rviz_plugins/Goal3DTool with base class type rviz::Tool does not exist. Declared types are  rviz/FocusCamera rviz/Interact rviz/Measure rviz/MoveCamera rviz/PublishPoint rviz/Select rviz/SetGoal rviz/SetInitialPose rviz_plugin_tutorials/PlantFlag`

`[Err] [InsertModelWidget.cc:403] Missing model.config for model "/opt/ros/noetic/share/turtlebot3_gazebo/models/turtlebot3_autorace"
[Err] [InsertModelWidget.cc:403] Missing model.config for model "/opt/ros/noetic/share/turtlebot3_gazebo/models/turtlebot3_autorace_2020"
[Err] [InsertModelWidget.cc:403] Missing model.config for model "/opt/ros/noetic/share/turtlebot3_gazebo/models/turtlebot3_common_meshes"
`

`[Err] [Material.cc:91] Unable to find texture[SVGMat.001 Base Color.png] in path[/home/fabio/catkin_ws/src/arena-bench/simulator_setup/worlds/map2/worlds]`

**Comment by AlexanderDavid** [comment]
@fabiodl Did you resolve this issue? We are interested in using this software but not if there are still a bunch of errors you are experiencing.

**Comment by fabiodl** [comment]
@AlexanderDavid No, I switched to https://github.com/NKU-MobFly-Robotics/local-planning-benchmark

**Comment by HareshKarnan** [comment]
I was able to get the package to compile successfully and roslaunch

**Comment by linhdoan8** [comment]
Hey all,thanks for your intrest in using our software. And sorry for experiencing those issues. While this repo should still be working fine, we have moved to a new repo https://github.com/Arena-Rosnav with more functionalities and simplified installation. The new repo contains all functions of arena-rosnav and arena-bench. We restructured the whole arena repos and you should be able to install and run without any issues. Issues will also be resolved much faster by our team. Please check it out!On 20 Jul 2023, at 05:51, Haresh ***@***.***> wrote:﻿
I was able to get the package to compile successfully and roslaunch
